keps/sig-api-machinery/3352-aggregated-discovery/README.md

```diff
@@ -748,11 +772,20 @@ term, we may want to require automated upgrade/rollback tests, but we
 are missing a bunch of machinery and tooling and can't do that now.
 -->
 
+n/a.
```

Comment:
Please expand.  I think n/a is appropriate here, but only because this API does not store data, so the state is recalculated on restart during the upgrade,downgrade,upgrade cycle.

Reply:
Updated